Pipfile

```diff
@@ -15,6 +15,7 @@ mypy = "*"
 
 [packages]
 launchable = {editable = true,path = "."}
+click = "~=7.1"
```

Comment:
As we discussed in Slack, Click v8.0.0 was released yesterday and some tests does not work with the version